services/management/postgresql.go

```diff
@@ -70,6 +70,20 @@ func (s *PostgreSQLService) Add(ctx context.Context, req *managementpb.AddPostgr
 		}
 		res.Service = invService.(*inventorypb.PostgreSQLService)
 
+		switch {
+		case req.AddNode != nil && req.AddNode.NodeType == inventorypb.NodeType_REMOTE_NODE:
```

Comment:
Now at the second sight it looks like it's better to check for `req.PmmAgentId == models.PMMServerAgentID`, because that's the reason why agents added via UI should be pull.
Sorry, for confusing you, again.

Reply:
here and in other places.